Forge.TreeWalker/src/TreeWalkerSession.cs

```diff
@@ -640,6 +640,7 @@ internal async Task PerformActionTypeBehavior(TreeNode treeNode, string treeNode
             CancellationToken token)
         {
             // Initialize values. Default infinite timeout. Default RetryPolicyType.None.
+            int maxRetryCount = treeAction.RetryPolicy != null ? treeAction.RetryPolicy.MaxRetry : 0; // Specific setting for RetryPolicyType.FixedCount
```

Comment:
>int maxRetryCount = treeAction.RetryPolicy != null ? treeAction.RetryPolicy.MaxRetry : 0; // Specific setting for RetryPolicyType.FixedCount [](start = 12, length = 140)

Let's place this below the line "TimeSpan waitTime = ..."

Also, let's shorthand the logic to this:
int maxRetryCount = treeAction.RetryPolicy?.MaxRetry ?? 0;

Reply:
Nice, didn't know about that shorthand.